src/bsp/src/stm32/src/SpiEsp.cpp

```diff
@@ -86,14 +104,20 @@ void SpiEsp::execute() {
         m_inboundHeader = (EspHeader::Header*)m_inboundMessage.m_data.data();
         if (m_inboundHeader->crc8 !=
             m_crc.calculateCRC8(m_inboundHeader, EspHeader::sizeBytes - 1)) {
-            m_logger.log(LogLevel::Error, "Received corrupted SPI ESP header");
+            m_logger.log(LogLevel::Debug, "Received corrupted SPI ESP header");
```

Comment:
Is this frequent so you setted it in debug?

Reply:
I added it so if you dev without a HiveSight, or without the ESP flashed with the right firmware, it will not pollute the logs.